### ISSUE TITLE
Unexpected class java.lang.RuntimeException

### ISSUE THREAD

**Issue by roger109z** [CCEmuX/CCEmuX#89]
```
CCEmuX version: unknown
OS name: Windows 10
OS version: 10.0
OS architecture: amd64
JRE vendor: Oracle Corporation
JRE version: 9.0.4
JRE architecture: 64

java.lang.RuntimeException: Classloader in use is not a URLClassLoader
	at net.clgd.ccemux.init.Launcher.loadPlugins(Launcher.java:184)
	at net.clgd.ccemux.init.Launcher.launch(Launcher.java:254)
	at net.clgd.ccemux.init.Launcher.main(Launcher.java:87)
```

**Comment by SquidDev** [comment]
I believe this hasn't been an issue for a while. When/where did you get this version?

**Comment by dmarcuse** [comment]
Yes, this was fixed a while ago with f171e5f2b802996d8ad4541770725c38c1dd5388. Make sure you're using the `revival` branch.
